src/components/InputTodo.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  state = {
+    title: '',
+  };
+
+  onChange = (e) => {
+    this.setState({
+      title: e.target.value,
+    });
+  };
```

Comment:
It would be nice if you declared another name to avoid confusion about onChange function.

Reply:
The name is provided from a tutorial. To keep the tutorial work more manageable and cleaner, I can finish the tutorial, then edit function names.